pkg/services/librarypanels/librarypanels_test.go

```diff
@@ -633,15 +634,15 @@ func toLibraryElement(t *testing.T, res libraryelements.LibraryElementDTO) libra
 			ConnectedDashboards: res.Meta.ConnectedDashboards,
 			Created:             res.Meta.Created,
 			Updated:             res.Meta.Updated,
-			CreatedBy: libraryelements.LibraryElementDTOMetaUser{
-				ID:        res.Meta.CreatedBy.ID,
+			CreatedBy: librarypanel.LibraryElementDTOMetaUser{
+				Id:        res.Meta.CreatedBy.Id,
 				Name:      res.Meta.CreatedBy.Name,
-				AvatarURL: res.Meta.CreatedBy.AvatarURL,
+				AvatarUrl: res.Meta.CreatedBy.AvatarUrl,
```

Comment:
Not sure, but I thought we had some convention to spell these as `ID` and `URL` in all caps in go files?

Reply:
we do... but the generated files do not follow it 😢